slr/basic_app/templates/basic_app/frontend_submission.html

```diff
@@ -1,11 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8">
+        <!-- Required meta tags always come first -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta http-equiv="x-ua-compatible" content="ie=edge">
+
+	
+
         <title>Front End Submission</title>
     </head>
     <body>
-        <h1>nothing to look at here.</h1>
-        <h2>At least not yet.</h2>
+        <nav  class="navbar navbar-light " >
+            <div class="container" style="background-color:cornsilk; color: grey;" >
+                
+					<a class="nav-link "  href="/index.html"> WSoC</a>
+					<a class="nav-link" href="/frontend.html">Front End</a>
+				<a class="nav-link" href="/backend.html">Back End</a>
+					<a class="nav-link active" href="#"> Front End Submission</a>
+             
+                </div>
+                </nav>
+
+        
+
+        
+       
+	<header class="header">
+		<div class="container">
+			
+				
+				<div class="col-12 col-sm align-self-center">
+					<img src="../static/basic_app/Logo-White-Header.png" class="img-fluid">
+				</div>
+				
+				
+		</div>
+    </header>
+    
+
+	<div class="container">
+        <div class="row row-header" >
+		<h1><center>FRONT END</center></h1>
+        </div >
+        <div class="row row-content" style="margin-top:50px;">
+            <h3 style="margin: 50px right; margin:50px left;"> Web Development </h3>
+           
+        </div>
+        <div>
+            <p style="margin: 90px right; margin-top:40px;"> This is my web-page. And I worked on its Front-end. </p>
+        </div>
+            
+        <div class="row row-content">
+        <div class="mr-50">
+        <img src="../static/basic_app/doggo-bread.jpg" style="height: 180px; width:220px; margin-left: 80%;;" alt="doggo-bread">
```

Comment:
You can utilize Django's features to allow working with templates, which could be considered as a upgrade from relative mapping to images, and other pages.
Look into [managing static files](https://docs.djangoproject.com/en/dev/howto/static-files/) via Django.